Makefile

```diff
@@ -580,7 +580,7 @@ generate-release-notes: $(RELEASE_NOTES_DIR)
 
 .PHONY: promote-images
 promote-images: $(KPROMO)
-	$(KPROMO) pr --project cluster-api-provider-vsphere --tag $(RELEASE_TAG) --reviewers "$(IMAGE_REVIEWERS)" --fork $(USER_FORK) --image cluster-api-provider-vsphere
+	$(KPROMO) pr --project capi-vsphere --tag $(RELEASE_TAG) --reviewers "$(IMAGE_REVIEWERS)" --fork $(USER_FORK) --image cluster-api-vsphere-controller
```

Comment:
Fixups so it looks in the right project for our stuff 

(`gcr.io/k8s-staging-capi-vsphere/cluster-api-vsphere-controller:main`, project is prefixed with k8s-staging)

I think this command will only actually work once we did one release manually so we have a YAML with existing images in the image promotion repo (currently this yaml is just empty)